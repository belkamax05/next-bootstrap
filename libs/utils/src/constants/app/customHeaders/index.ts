const customHeaders = {
  /** Used to pass pathname on server-side */
  currentPath: 'x-current-path',
} as const;

export default customHeaders;
