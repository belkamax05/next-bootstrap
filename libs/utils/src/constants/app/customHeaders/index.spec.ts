import customHeaders from '.';

describe('should correctly import constants', () => {
  it('should have correct customHeaders', () => {
    expect(customHeaders).toMatchSnapshot();
  });
});
