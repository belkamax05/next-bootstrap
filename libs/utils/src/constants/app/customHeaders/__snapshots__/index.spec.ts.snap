// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly import constants should have correct customHeaders 1`] = `
Object {
  "currentPath": "x-current-path",
}
`;
